draft-cardwell-ccwg-bbr.xml

```diff
@@ -1067,7 +1067,7 @@ sustain its achievable throughput.</t>
       random_int_between(0, 1); /* 0 or 1 */
     /* Decide the random wall clock bound for wait: */
     BBR.bw_probe_wait =
-      2 + random_float_between(0.0, 1.0) /* 0..1 sec */ secs
```

Comment:
It seems like the removal of "secs" is the contribution here?

Can you please rebase your commit onto the latest draft commits, and squash your commits into a single commit that makes this change? Thanks!

Reply:
Ok